src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -213,7 +213,8 @@ internal BinaryClassificationTrainers(BinaryClassificationCatalog catalog)
         /// <summary>
         /// Run cross-validation over <paramref name="numberOfFolds"/> folds of <paramref name="data"/>, by fitting <paramref name="estimator"/>,
         /// and respecting <paramref name="samplingKeyColumnName"/> if provided.
-        /// Then evaluate each sub-model against <paramref name="labelColumnName"/> and return metrics.
+        /// Then evaluate each sub-model against <paramref name="labelColumnName"/> and return <see cref="BinaryClassificationMetrics"/>, which
```

Comment:
>return [](start = 88, length = 6)

A list of BinaryClassificationMetrics. Or, "return a BinaryClassificationMetrics object for each sub-model".